joblib/parallel.py

```diff
@@ -129,12 +190,238 @@ def get_active_backend(prefer=None, require=None, verbose=0):
+
+        In addition, if the `dask` and `distributed` Python packages are
+        installed, it is possible to use the 'dask' backend for better
+        scheduling of nested parallel calls without over-subscription and
+        potentially distribute parallel calls over a networked cluster of
+        several hosts.
+
+        It is also possible to use the distributed 'ray' backend for
+        distributing the workload to a cluster of nodes. To use the 'ray'
+        joblib backend add the following lines::
+
+        >>> from ray.util.joblib import register_ray  # doctest: +SKIP
+        >>> register_ray()  # doctest: +SKIP
+        >>> with parallel_backend("ray"):  # doctest: +SKIP
+        ...     print(Parallel()(delayed(neg)(i + 1) for i in range(5)))
+        [-1, -2, -3, -4, -5]
+
+        Alternatively the backend can be passed directly as an instance.
+
+    n_jobs : int, default=None
+        The maximum number of concurrently running jobs, such as the number
+        of Python worker processes when backend="multiprocessing"
+        or the size of the thread-pool when backend="threading".
+        If -1 all CPUs are used. If 1 is given, no parallel computing code
+        is used at all, which is useful for debugging. For n_jobs below -1,
+        (n_cpus + 1 + n_jobs) are used. Thus for n_jobs = -2, all
+        CPUs but one are used.
+        None is a marker for 'unset' that will be interpreted as n_jobs=1.
+
+    verbose : int, default=0
+        The verbosity level: if non zero, progress messages are
+        printed. Above 50, the output is sent to stdout.
+        The frequency of the messages increases with the verbosity level.
+        If it more than 10, all iterations are reported.
+
+    temp_folder : str, default=None
+        Folder to be used by the pool for memmapping large arrays
+        for sharing memory with worker processes. If None, this will try in
+        order:
+
+        - a folder pointed by the JOBLIB_TEMP_FOLDER environment
+            variable,
+        - /dev/shm if the folder exists and is writable: this is a
+            RAM disk filesystem available by default on modern Linux
+            distributions,
+        - the default system temporary folder that can be
+            overridden with TMP, TMPDIR or TEMP environment
+            variables, typically /tmp under Unix operating systems.
+
+    max_nbytes int, str, or None, optional, default='1M'
+        Threshold on the size of arrays passed to the workers that
+        triggers automated memory mapping in temp_folder. Can be an int
+        in Bytes, or a human-readable string, e.g., '1M' for 1 megabyte.
+        Use None to disable memmapping of large arrays.
+
+    mmap_mode: {None, 'r+', 'r', 'w+', 'c'}, default='r'
+        Memmapping mode for numpy arrays passed to workers. None will
+        disable memmapping, other modes defined in the numpy.memmap doc:
+        https://numpy.org/doc/stable/reference/generated/numpy.memmap.html
```

Comment:
It could be nice to make this link clickable.

Reply:
it is on joblib's doc, see https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html